clang-tools-extra/docs/clang-tidy/checks/readability/use-std-min-max.rst

```diff
@@ -0,0 +1,33 @@
+.. title:: clang-tidy - readability-use-std-min-max
+
+readability-use-std-min-max
+===========================
+
+Replaces certain conditional statements with equivalent ``std::min`` or ``std::max`` expressions, 
```

Comment:
Please follow 80-characters limit.

Reply:
Line is still longer than 79 characters. Same below.

Reply:
Oh sorry I misunderstood it to keep that in 80 words limit total, but got it now its 80 characters for each line am I right?

Reply:
I have fixed it aswell